src/apps/material/material.tsx

```diff
@@ -17,8 +17,14 @@ const Material: React.FC<MaterialProps> = ({ pid, workId }) => {
     return <div>Loading...</div>;
   }
 
+  if (!data?.work) {
+    return <div>No work data</div>;
```

Comment:
DDF wont accept that we write an english string like this.
If we choose this solution I think the text should be translatable 🎉